src/app/login/page.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
```

Comment:
Why are there 2 login pages.
delete the one that is not active and leave the one.